templates/boot-fastboot/meson-g12a-sei510.jinja2

```diff
@@ -0,0 +1,17 @@
+- test:
+    namespace: tlxc
+    timeout:
+      minutes: 30
+    definitions:
+    - from: inline
+      name: sei510-images
+      path: inline/sei510-images.yaml
+      repository:
+        metadata:
+          description: sei510 images
+          format: Lava-Test Test Definition 1.0
+          name: sei510-images
+        run:
+          steps:
+          - cd /lava-lxc
+          - mkbootimg --kernel {{ kernel_image }} --second {{ dtb_short }} --ramdisk rootfs.cpio.gz --cmdline "console=ttyAML0,115200 consoleblank=0 earlycon" --kernel_offset 0x01080000 --ramdisk_offset 0x6000000 --second_offset 0x1000000 -o boot.img
```

Comment:
It would be nice to template this a little more, having a standard `mkbootimg` template and getting the cmdline and various addresses from a device-type specific template.

Reply:
To do this we have to modify the lava-v2-jobs-from-api.py code to take the variables from the device-type and put them in the mkbootimg template. 
Proceed this way ?

Reply:
Yes.

Reply:
Well, this should be coming from `kernelci.configs.DeviceType`, to have some parameters like for the TestPlans (this is what I explained in a comment in the main discussion for this PR).